src/main/resources/application.properties

```diff
@@ -1 +1,13 @@
+server.port=8080
```

Comment:
this is by default 8080, you don't need to write explicitly, if you are using default port

Reply:
remove